trusted.ci.jenkins.io.tf

```diff
@@ -21,32 +21,59 @@ data "azurerm_subnet" "trusted_ephemeral_agents" {
   resource_group_name  = data.azurerm_resource_group.trusted.name
   virtual_network_name = data.azurerm_virtual_network.trusted.name
 }
-resource "azurerm_resource_group" "trusted_ci_jenkins_io_ephemeral_agents" {
-  name     = "jenkinsinfra-trusted-ephemeral-agents"
-  location = var.location
-  tags     = local.default_tags
+####################################################################################
+## Resources for the Controller VM
+####################################################################################
+module "trusted_ci_jenkins_io" {
+  source = "./.shared-tools/terraform/modules/azure-jenkins-controller"
+
+  controller_short_hostname    = "trusted.ci"
+  location                     = data.azurerm_virtual_network.trusted.location
+  admin_username               = local.admin_username
+  admin_ssh_publickey          = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQC5K7Ro7jBl5Kc68RdzG6EXHstIBFSxO5Da8SQJSMeCbb4cHTYuBBH8jNsAFcnkN64kEu+YhmlxaWEVEIrPgfGfs13ZL7v9p+Nt76tsz6gnVdAy2zCz607pAWe7p4bBn6T9zdZcBSnvjawO+8t/5ue4ngcfAjanN5OsOgLeD6yqVyP8YTERjW78jvp2TFrIYmgWMI5ES1ln32PQmRZwc1eAOsyGJW/YIBdOxaSkZ41qUvb9b3dCorGuCovpSK2EeNphjLPpVX/NRpVY4YlDqAcTCdLdDrEeVqkiA/VDCYNhudZTDa8f1iHwBE/GEtlKmoO6dxJ5LAkRk3RIVHYrmI6XXSw5l0tHhW5D12MNwzUfDxQEzBpGK5iSfOBt5zJ5OiI9ftnsq/GV7vCXfvMVGDLUC551P5/s/wM70QmHwhlGQNLNeJxRTvd6tL11bof3K+29ivFYUmpU17iVxYOWhkNY86WyngHU6Ux0zaczF3H6H0tpg1Ca/cFO428AVPw/RTJpcAe6OVKq5zwARNApQ/p6fJKUAdXap+PpQGZlQhPLkUbwtFXGTrpX9ePTcdzryCYjgrZouvy4ZMzruJiIbFUH8mRY3xVREVaIsJakruvgw3b14oQgcB4BwYVBBqi62xIvbRzAv7Su9t2jK6OR2z3sM/hLJRqIJ5oILMORa7XqrQ=="
+  dns_zone_name                = azurerm_private_dns_zone.trusted.name
+  dns_resourcegroup_name       = data.azurerm_resource_group.trusted.name
+  controller_subnet_id         = data.azurerm_subnet.trusted_ci_controller.id
+  controller_data_disk_size_gb = 128
+  controller_vm_size           = "Standard_D2as_v5"
+  default_tags                 = local.default_tags
+
+  controller_resourcegroup_name = "jenkinsinfra-trusted-ci-controller"
+  controller_vm_name            = ""
```

Comment:
why is this set to an empty string?

Reply:
https://github.com/jenkins-infra/azure/pull/452